_docs/_latest/develop/dev/setup.md

```diff
@@ -155,41 +205,45 @@ try the following:
 
 ------------------
 
```

Comment:
Are you sure about this?  I was able to fix just by adding a dummy my_config.h file to the directory where #includes looks from.  But if this works then this is a better solution.  I had read that python expects mysql to need the my_config even though this is no longer the case hence why the dummy file works

Reply:
Yes, I am sure about this, this was the only thing I had to try to fix this problem.